docs/tutorial/t-deploy.md

```diff
@@ -84,17 +84,59 @@ unit-kafka-1:
     started: 2023-04-25 09:02:59 +0000 UTC
 ```
 
-Apache Kafka ships with `bin/*.sh` commands to do various administrative tasks, e.g `bin/kafka-config.sh` to update cluster configuration, `bin/kafka-topics.sh` for topic management, and many more! The Kafka Charmed Operator provides these commands to administrators to easily run their desired cluster configurations securely with SASL authentication, either from within the cluster or as an external client.
+Providing you the `username` and `password` of the Kafka cluster admin user. 
 
-If you wish to run a command from the cluster, in order to (for example) list the current topics on the Kafka cluster, you can run:
+**IMPORTANT** Note that when no other application is related to Kafka, the cluster is secured-by-default and external listeners (binded to port 9092) are disabled, thus preventing any external incoming connection. 
+
+Nevertheless, it is still possible to run a command from within the Kafka cluster using the internal listeners in place of the external ones. 
+The internal endpoints can be constructed by replacing the 19092 port in the `bootstrap.servers` returned in the output above, e.g. 
+
+```shell
+INTERNAL_SERVERS=$(juju run kafka/leader get-admin-credentials | grep "bootstrap.servers" | cut -d "=" -f2 | sed -s "s/\:9092/:19092/g")
 ```
-BOOTSTRAP_SERVERS=$(juju run-action kafka/leader get-admin-credentials --wait | grep "bootstrap.servers" | cut -d "=" -f 2)
-juju ssh kafka/leader 'charmed-kafka.topics --bootstrap-server $BOOTSTRAP_SERVERS --list --command-config /var/snap/charmed-kafka/common/client.properties'
+
+Once you have fetched the `INTERNAL_SERVERS`, log in into one of the Kafka container in one of the units
+
+```shell
+juju ssh kafka/leader /bin/bash
```

Comment:
**nitpick:** To make life easier, I'd suggest to change to `juju ssh kafka/leader sudo -i`. As oftentimes they need `sudo` later.